.env.example

```
@@ -12,6 +12,13 @@ EHRBASE_PASSWORD=
 # CORS origins - for API service (JSON array)
 CORS_ORIGINS=["http://localhost:5173"]
 
+# Terminology server (Snowstorm Lite) - for API service
+TERMINOLOGY_SERVER_URL=http://localhost:8081/fhir
+TERMINOLOGY_SERVER_TIMEOUT=30
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TERMINOLOGY_SERVER_TIMEOUT key should go before the TERMINOLOGY_SERVER_URL key

(UnorderedKey)